### PR TITLE
Auth: None basic role update docs

### PR DESCRIPTION
**Pull request by eleijonmarck**
**What is this feature?**
Adds None basic role to the documentation

part of none basic role rollout - https://github.com/grafana/identity-access-team/issues/134

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by IevaVasiljeva — COMMENTED** [review]
Some other docs that might need updating:
* https://grafana.com/docs/grafana/latest/administration/user-management/manage-org-users/#invite-a-user-to-join-an-organization
* https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/generic-oauth/#map-user-organization-role and other IdP integration docs (assuming that role None works with role sync?);
* https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/enterprise-configuration/#org_mapping
* https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/#anonymous-authentication

**Comment by eleijonmarck** [comment]
il also review these:

-[ ] https://grafana.com/docs/grafana/next/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/
- [ ] https://grafana.com/docs/grafana/next/administration/roles-and-permissions/access-control/plan-rbac-rollout-strategy/#review-basic-role-and-fixed-role-definitions
- [ ] https://grafana.com/docs/grafana/next/administration/roles-and-permissions/access-control/plan-rbac-rollout-strategy/#permissions-scenarios
- [ ] https://grafana.com/docs/grafana/next/administration/roles-and-permissions/access-control/manage-rbac-roles/
- [ ] https://grafana.com/docs/grafana/next/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/
- [ ] https://grafana.com/docs/grafana/next/setup-grafana/configure-security/planning-iam-strategy/

File: docs/sources/administration/roles-and-permissions/access-control/plan-rbac-rollout-strategy/index.md [hunk 1]
```diff
@@ -117,6 +117,11 @@ If you have a use case that you'd like to share, feel free to contribute to this
 1. Add dashboards to each folder.
 1. Use folder permissions to add US-based users as Editors to the `US` folder and assign EU-based users as Editors to the `EU` folder.
 
+### Assign a user specific set of roles
+
+1. Create a user with the `No Basic Role` role.
```
Comment (IevaVasiljeva) [inline docs/sources/administration/roles-and-permissions/access-control/plan-rbac-rollout-strategy/index.md:122]: Nit: not sure how to best phrase this, but  "`No Basic Role` role" sounds weird to me.
```suggestion
1. Create a user with the `No Basic Role` selected under organization roles.
```

File: docs/sources/administration/user-management/manage-org-users/index.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ If you have [server administrator]({{< relref "../../roles-and-permissions/#graf
 
 ## Invite a user to join an organization
 
-When you invite users to join an organization, you assign the **Admin**, **Editor**, or **Viewer** role which controls user access to the dashboards and data sources owned by the organization. Users receive an email that prompts them to accept the invitation.
+When you invite users to join an organization, you assign the **Admin**, **Editor**, **Viewer**, or **None** role which controls user access to the dashboards and data sources owned by the organization. Users receive an email that prompts them to accept the invitation.
```
Comment (IevaVasiljeva) [inline docs/sources/administration/user-management/manage-org-users/index.md:77]: Nit: I think we're referring to frontend workflows in this guide, and from the frontend we call this "No basic role"
```suggestion
When you invite users to join an organization, you assign the **Admin**, **Editor**, or **Viewer** role, or select **No basic role**. Organization roles control user access to resources, such as dashboards and data sources, owned by the organization. Users receive an email that prompts them to accept the invitation.
```

**Review by IevaVasiljeva — APPROVED** [review]
Looks good! Left a few suggestions in the comments

Have we only implemented role-mapping to `None` for generic OAuth? Just wanted to check if there are any more IdP integration docs that need updating.

**Pull request merged**